src/hand_gesture_recognition/hand_classifier.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+import numpy as np
+import tensorflow as tf
+import cv2
+
+
+class HandClassifier:
+    PREPROCESSED = False
+
+    def __init__(self, models_dir):
+        try:
+            self.model = tf.keras.models.load_model(models_dir + "trained_palm_fist.h5")
+        except OSError as e:
+            raise e
+
+        self.input_shape = self.model.layers[0].input_shape
+        print("Hi I'm classifier")
```

Comment:
Change this to something like: "Classifier loaded successfully"

Reply:
Yeah Yeah, it was for testing too